hyper/hyper/restore.go

```diff
@@ -0,0 +1,14 @@
+package hyper
+
+import (
+	"os"
+)
+
+func (h *Hyper) Restore(path string) error {
+	_, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
```

Comment:
Is this really needed? Or can we handle the libvirt error to find a not found or permission errors?

Reply:
can't be handled by libvirt.